docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.slick.scss

```diff
@@ -0,0 +1,214 @@
+      }
+
+      &:hover {
+        &::before {
+          transform: translateX(2px);
+        }
+      }
+    }
+
+    &.slick-prev,
+    &.slick-next {
+      @include grid-media-min('md') {
+        @include hb-traditional {
+          bottom: calc(50% - #{hb-calculate-rems(30px)});
+        }
+
+        @include hb-themes(('colorful', 'airy')) {
+          bottom: calc(50% - #{hb-calculate-rems(21px)});
+        }
+
+        // styles specific to the main content area are applied by targeting
+        // the sidebar classes. We can use these classes to move the slick dots.
+        :not(.hs-full-width).hb-three-column--no-sidebar &,
+        .hb-three-column--one-sidebar &,
+        .hb-three-column--two-sidebar & {
+          bottom: calc(50% - #{hb-calculate-rems(46px)});
+        }
+      }
+    }
+  }
+}
+
+// dots aka bars
+@mixin slick-dots {
+  .slick-dots {
+    @include visually-hidden;
+
+    @include grid-media-min('md') {
+      display: inline-block;
+      clip: inherit;
+      height: auto;
+      width: auto;
+      position: absolute;
+      bottom: 0;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      margin: 0;
+      padding: 0;
+      list-style-type: none;
+
+      @include hb-themes(('colorful', 'airy')) {
+        bottom: hb-calculate-rems(16px);
+      }
+
+      // styles specific to the main content area are applied by targeting
+      // the sidebar classes. We can use these classes to move the slick dots.
+      :not(.hs-full-width).hb-three-column--no-sidebar &,
+      .hb-three-column--one-sidebar &,
+      .hb-three-column--two-sidebar & {
+        bottom: hb-calculate-rems(-32px);
+      }
+
+      li {
+        display: inline-block;
+        line-height: 25%;
+
+        button {
+          @include hb-unbuttonize;
+          @include hb-global-color('background-color', 'gray');
+          padding: 0 hb-calculate-rems(22px);
+          margin: 0 hb-calculate-rems(8px);
+          font-size: 0; // hide text
+          transition: hb-transition(transform);
+
+          @include hb-themes(('colorful', 'airy')) {
+            border-radius: hb-calculate-rems(5px);
+          }
+
+          &:hover {
+            transform: scale(0.9);
+
+            @include hb-traditional {
+              @include hb-global-color('background-color', 'black');
+            }
+
+            @include hb-themes(('colorful', 'airy')) {
+              @include hb-pairing-color('background-color', 'secondary');
+            }
+          }
+        }
+
+        // active button representing the slide currently on display
+        &.slick-active button {
+          @include hb-traditional {
+            @include hb-global-color('background-color', 'black');
+          }
+
+          @include hb-themes(('colorful', 'airy')) {
+            @include hb-pairing-color('background-color', 'secondary');
+          }
+
+          &:hover {
+            transform: none;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Mixin additions were fantastic ideas for this and should make things easy to update going forward!